project/Dependencies.scala

```diff
@@ -238,7 +238,7 @@ object Dependencies {
         "com.google.auth" % "google-auth-library-oauth2-http" % "0.20.0", // BSD 3-clause
         // pull in Akka Discovery for our Akka version
         "com.typesafe.akka" %% "akka-discovery" % AkkaVersion
-      ) ++ Silencer
```

Comment:
[question] Curious as to why the `Silencer` dep usage was moved to build.sbt?

Reply:
My idea was that it is not a dependency. It's just build infrastructure. I'll move it back.